src/main/menu.ts

```diff
@@ -46,10 +45,33 @@ export default class MenuBuilder {
       ]
     };
 
-    return [EditMenu, this.buildDefaultTemplate()];
+    return [EditMenu, this.buildFileTemplate(), this.buildHelpTemplate()];
+  }
+  buildFileTemplate() {
+    const fileTemplate = {
+      label: "File",
+      submenu: [
+        {
+          label: "Open Project...",
+          click: () => {
+            //Fire a message to the renderer process.
+            //Handled in stores/Workspace
+            this.mainWindow.webContents.send("openProject");
+          }
+        },
+        {
+          label: "Exit",
+          click: () => {
+            this.mainWindow.close();
```

Comment:
If this should to close the app, shall we use `app.quit()`? This would close the window but the app would be still running.

Reply:
From what I can tell, both exit the application. This is the same as clicking the x in the top right. And with this way, the processes are still being killed. Happy to change though.

Reply:
I see. Are you on windows or mac? I'm not sure whether this would work on a mac since closing the window minimizes the app but then we have to explicitly quit the app to exit. But let's leave this for now and see 👍

Reply:
`app.quit` is probably nicer. I'll switch it. Let me know if you have any issues on mac. I'm on windows atm but can test on mac too if necessary.

Reply:
Thanks. That's fine. I'll spin up my VM and test it a bit later.